examples/jira-multiproject.gitbugtraq

```diff
@@ -0,0 +1,11 @@
+# An example content of .gitbugtraq to put in the repository root directory.
+# It could instead be added as an additional section to $GIT_DIR/config.
+# (note that '\' need to be escaped).
+# This shows how to use detection for only specific projects in  JIRA
+[bugtraq "jira-mariadb"]
+
+; use only specific projects from https://jira.mariadb.org/secure/BrowseProjects.jspa?selectedCategory=all&selectedProjectType=all
```

Comment:
Is there a specific reason why you are using ";" instead of "#" as comment char? If not, please change to "#" to be consistent with all other comments in the examples.

Reply:
Fixed.